superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -165,6 +213,17 @@ export default class FilterableTable extends PureComponent {
     this.setState({ sortBy, sortDirection });
   }
 
+  addJsonModal(node, jsonObject, jsonString) {
+    return (
+      <ModalTrigger
```

Comment:
I'm not sure if perf matters here as I think we're always using `react-virtualized` in this context (only rendering the rows in the viewport), but I think this will build all of the modals and related `<JSONTree />`. @betodealmeida has been working on data preview perf recently and may have more intuition as to whether this matters.

An alternate approach would be to build / render the modal on only when clicked/used.

Reply:
Looking at the DOM and React devtools, it doesn't seem like anything gets rendered prior to clicking on the cell. The react structure has a ModalTrigger and an empty Modal element and that's it. I think we're probably fine here.